doc/api/dgram.md

```diff
@@ -159,7 +159,7 @@ added: v0.11.14
 -->
 
 * `options` {Object} - Required. Supports the following properties:
-  * `port` {Number} - Required.
+  * `port` {Number} - Optional.
```

Comment:
Unfortunately, socket.bind(port) and socket.bind(options) have cut-n-pasted docs, so you need to change in both places (or document the number variant by referring to the options variant).